### PR TITLE
updated url bar onion indicators

### PR DESCRIPTION
**Pull request by JudeUk**
updated the url bar onion indicators in Tor documentation

### CODE REVIEW

**Review by gusgustavo — CHANGES_REQUESTED** [review]
Hi, 

1. Instead of using .svg, could you use .png?
2. Generate png with the same size that we have now?

**Comment by JudeUk** [comment]
Hi @gusgustavo i have done as requested, kindly review thanks.

**Review by gusgustavo — APPROVED** [review]
Hi, thanks for working on this issue. 
I think it's ready to merge when we release Tor Browser 9.5.

**Comment by JudeUk** [comment]
great thanks for the review @gusgustavo

File: content/onionservices/onionservices-5/contents.lr [hunk 1]
```diff
@@ -7,15 +7,18 @@ seo_slug: address-bar-onion-icons
...
 When browsing an Onion Service, Tor Browser displays different onion icons in the address bar indicating the security of the current webpage.
 
-![Image of a green onion](/static/images/green-onion.png)
-A green onion means:
-- The Onion Service is served over HTTP, or HTTPS with a self-signed certificate.
+![Image of a black onion](/static/images/black-onion.png)
+A black onion means:
+- The Onion Service is served over HTTP, or HTTPS with a CA-Issued certificate.
+- The Onion Service is served over HTTPS with a Self-Signed certificate.
+- The Onion Service is served over HTTPS with EV.
```
Comment (asn-d6) [inline content/onionservices/onionservices-5/contents.lr:14]: What's the difference of "- The Onion Service is served over HTTPS with EV." and "- The Onion Service is served over HTTP, or HTTPS with a CA-Issued certificate.". I think the latter also includes the former.

**Comment by acatarineu** [comment]
We could maybe use "onion" directly instead of "black onion"? Before it was green, which has good security connotations, but I think the new color (gray/black) is neutral so we can just omit it.

**Comment by asn-d6** [comment]
> We could maybe use "onion" directly instead of "black onion"? Before it was green, which has good security connotations, but I think the new color (gray/black) is neutral so we can just omit it.

Yes agreed. I also found the "black onion" thing a bit weird in terms of feeling produced.

File: content/onionservices/onionservices-5/contents.lr (hunk 1 above)
Comment (gusgustavo) [reply content/onionservices/onionservices-5/contents.lr:14]: Fixed!

**Comment by gusgustavo** [comment]
I cherry-picked, added the fixes and merged. Thanks!

**Pull request closed without merging**